packages/commonwealth/client/scripts/views/pages/discussions/ThreadCard/ThreadOptions/AdminActions/AdminActions.tsx

```diff
@@ -273,9 +299,10 @@ export const AdminActions = ({
         <PopoverMenu
           className="AdminActions compact"
           menuItems={[
-            ...(hasAdminPermissions ||
-            isThreadAuthor ||
-            (isThreadCollaborator && !thread.readOnly)
+            ...(thread.archivedAt === null &&
```

Comment:
i feel like `!thread.archivedAt` is safer, curious for your thoughts

Reply:
It should either indicate a date or be `null`, hence the specificity.